### PR TITLE
feat: add test case

### PR DESCRIPTION
**Pull request by demohi**
We use node test/*.js now , It is a wrong way, because  it only run the first js file. So some errors will not be found in the CI process.

* use jest as test framework 
* test file named with *.test.js

### CODE REVIEW

**Comment by demohi** [comment]
@yorkie  after you add codecov integration, I will add the code coverage reports.

**Comment by yorkie** [comment]
The codecov has been installed for this repository, thanks for the awesome works, we will merge this if the CI is green :)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[Makefile]
+indent_style = tab
```
Comment (yorkie) [inline .editorconfig]: This file is not necessary

File: .travis.yml [hunk 2]
```diff
@@ -18,4 +18,7 @@ install:
   - npm install
 script:
   - echo $LD_LIBRARY_PATH
-  - npm test
+  - npm test-cov
+after_script:
+  - npm install codecov
+  - ./node_modules/.bin/codecov
```
Comment (yorkie) [inline .travis.yml]: move this script to package.json and use `npm run` to instead :)

File: package.json [hunk 3]
```diff
@@ -32,5 +33,8 @@
     "protobufjs": "6.8.0",
     "tar-fs": "^1.15.3",
     "unzip": "^0.1.11"
+  },
+  "devDependencies": {
+    "jest": "^20.0.4"
```
Comment (yorkie) [inline package.json]: Remove the ^

File: test/ops/base64.test.js [hunk 4]
```diff
@@ -17,8 +17,9 @@ function assertDecodeBase64(actual) {
...
-assertEncodeBase64('foobar');
-assertEncodeBase64('yorkie are working hard for tensorflow');
-assertDecodeBase64('beep boop');
-assertDecodeBase64('TensorFlow Node.js provides idiomatic JavaScript language bindings');
+test('op.base64', () => {
+  assertEncodeBase64('foobar');
+  assertEncodeBase64('yorkie are working hard for tensorflow');
+  assertDecodeBase64('beep boop');
+  assertDecodeBase64('TensorFlow Node.js provides idiomatic JavaScript language bindings');    
+})
```
Comment (yorkie) [inline test/ops/base64.test.js]: You are miss the semi-colon :)

File: test/tensor.test.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+const tf = require('../');
+
+test('tensor', () => {
+  const tensor = tf.tensor(Buffer.from([1, 2]), tf.dtype.int8, [2]);
+  assert.deepEqual(tensor.getViewData(), [1, 2]);
+
+  const text = 'foobar and yorkie are good friends';
+  const tensorTxt = tf.tensor(text);
+  assert.equal(tensorTxt.getViewData(), text);
+})
```
Comment (yorkie) [inline test/tensor.test.js]: Ditto

File: .editorconfig [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (yorkie) [inline .editorconfig]: Remove this file and add the .editorconfig to .gitignore, this file is user-specific, so let's do this.

File: package.json [hunk 7]
```diff
@@ -4,7 +4,8 @@
   "description": "TensorFlow Node.js provides idiomatic JavaScript language bindings and a high layer API for Node.js users.",
   "main": "lib/index.js",
   "scripts": {
-    "test": "node test/*.js",
+    "test": "jest",
+    "test-cov": "npm run test -- --coverage && npm install codecov && ./node_modules/.bin/codecov",
```
Comment (yorkie) [inline package.json]: does the empty tag `--` is unnecessary?

**Comment by yorkie** [comment]
@demohi Codecov is reported as: Unable to compare commits because the base of the compare did not upload a coverage report. Could we fix this?

File: package.json (hunk 7 above)
Comment (demohi) [inline package.json]: it is necessary for pass through the input 'coverage'

**Comment by demohi** [comment]
@yorkie  I think  the reason is  run code coverage first time , you can ignore it

File: .editorconfig (hunk 6 above)
Comment (demohi) [inline .editorconfig]: ok

**Comment by yorkie** [comment]
@demohi Could you please rebase again :)

**Comment by demohi** [comment]
@yorkie  done

**Comment by yorkie** [comment]
Rebased with small correction, thank you @demohi

**Pull request merged**